prometheus_client/multiprocess.py

```diff
@@ -7,11 +7,23 @@
 import json
 import os
 
+try:
+    from functools import lru_cache
+except ImportError:
+    # `lru_cache` is not available on Python 2;
+    # we simply don't get the caching benefits on Python 2, then.
+    def lru_cache(maxsize=128):
```

Comment:
Considering this is not actually LRU, hand implementing it is fine and will give us 2.x support

Reply:
Sure thing. I'll do that when I get back to the office tomorrow. I suspect using a simple dict cache will be faster on 3.x too, I just wanted to use the included batteries to begin with.

Reply:
Done in 5d0f877.